harfbuzz/Cargo.toml

```diff
@@ -20,3 +20,4 @@ default-features = false
 default = ["build-native-harfbuzz", "build-native-freetype"]
 build-native-harfbuzz = ["harfbuzz-sys/build-native-harfbuzz"]
 build-native-freetype = ["harfbuzz-sys/build-native-freetype"]
+use-pkg-config-harfbuzz = ["harfbuzz-sys/use-pkg-config-harfbuzz"]
```

Comment:
What does it mean when `build-native-freetype` and `build-native-harfbuzz` aren't active?

Reply:
Not sure about the `build-native-freetype`, but currently, if you `cargo test --no-default-features`, it doesn't link against any `harfbuzz` libs and so it fails to link with undefined symbols.